src/aspire/estimation/__init__.py

```diff
@@ -75,7 +77,7 @@ def src_backward(self):
             batch_mean_b = self.src.im_backward(im, i) / self.n
             mean_b += batch_mean_b.astype(self.as_type)
 
-        res = self.basis.evaluate_t(mean_b)
+        res = self.basis.evaluate_t(mean_b.T) # RCOPT
```

Comment:
I played around with this a little, but I haven't been satisfied with it yet.  Its not just for the unit tests.. there is also the check in `tutorials/examples.cov3d_simulation.py`.  I think this might require digging into `kernel.py`.  If what we are doing here is not acceptable short term, I can try to look into it immediately.  If we can live with it, I can add to the list. Let me know, thanks.

Reply:
That's puzzling. We can live with it for now, though.

Reply:
This is on the todo list now. I don't think it is worth plugging up the PRs for this atm. Having worked in/around the fb_3d code this morning, this seems like it is part of a larger collection of issues that should probably not be rushed out.

Reply:
Ok. I suspect this is related to the `covar_est = covar_est.T` issue discussed below. Fine to leave this for now and look at it later.